src/Pickles/Pickles/ObjectModel/Mapper.cs

```diff
@@ -260,7 +260,14 @@ public Feature MapToFeature(G.GherkinDocument gherkinDocument)
 
             foreach (var tag in gherkinDocument.Feature.Tags)
             {
-                feature.AddTag(this.MapToString(tag));
+                var stringTag = this.MapToString(tag);
+                var hideTags = string.IsNullOrEmpty(configuration.HideTags)?null:configuration.HideTags.Split(';');
+                if (hideTags != null && hideTags.Any(t => t.Equals($"@{stringTag}", StringComparison.InvariantCultureIgnoreCase)))
```

Comment:
The more crucial thing is that you are doing the filtering of the tags only on the feature level. Scenarios will still show the tag you are trying to hide.

If we introduce this feature, I would expect the tags to be hidden from scenarios as well.